Assets/00_Content/Scripts/Interactables/Table.cs

```diff
@@ -76,23 +76,20 @@ public class Table : Interactable {
 		}
 
 		private void StartRepairing(GameObject player, RepairTool tool) {
-			if (Tavern.Instance != null && RoundController.Instance != null)
-				Tavern.Instance.SpendsMoney(RoundController.Instance.CurrentDifficulty.tableRepairCost);
+			if (Tavern.Instance != null && RoundController.Instance != null) {
 
-			float repairTime = RoundController.Instance != null
-				? RoundController.Instance.CurrentDifficulty.tableRepairTime
-				: 5f;
-			tool.BeginRepairing(repairTime, transform.position + new Vector3(0, 3f, 0));//event
-			tool.RepairDone += HandleRepairDone;
+				float repairTime = RoundController.Instance != null
+					? RoundController.Instance.CurrentDifficulty.tableRepairTime
+					: 5f;
+				tool.BeginRepairing(repairTime, transform.position + new Vector3(0, 3f, 0)); //event
+				tool.RepairDone += HandleRepairDone;
 
-			player.GetComponent<PlayerMovement>().CanMove = false;
+				player.GetComponent<PlayerMovement>().CanMove = false;
+			}
 		}
 
 		private void EndRepairing(GameObject player, RepairTool tool) {
-			if (tool.IsRepairing) {
-				if (Tavern.Instance != null && RoundController.Instance != null)
-					Tavern.Instance.EarnsMoney(RoundController.Instance.CurrentDifficulty.tableRepairCost);
-			}
+			if (tool.IsRepairing)
 
 			tool.EndRepairing();
 			tool.RepairDone -= HandleRepairDone;
```

Comment:
The `if (tool.IsRepairing)` here attaches to the `tool.EndRepairing();` line below. You should probably remove the if.